layouts/community/ergodox/adnw_p_u_q/keymap.c

```diff
@@ -0,0 +1,359 @@
+#define PERMISSIVE_HOLD // tab/hold-Keys should work better with that
```

Comment:
This should be in your config.h file, actually. 

And you should have this at the top: 
```c
#include QMK_KEYBOARD_H
#include "keymap_german.h"
```